cluster/network.go

```diff
@@ -204,12 +208,15 @@ func (c *Cluster) doCalicoDeploy(ctx context.Context, data map[string]interface{
 		FlexVolImg:       c.SystemImages.CalicoFlexVol,
 		RBACConfig:       c.Authorization.Mode,
 		NodeSelector:     c.Network.NodeSelector,
+		OperatorImage:    c.SystemImages.TigeraOperator,
+		Kubectl:          c.Services.KubeAPI.Image, // Use same image that is used in GetAddonsExecuteJob for doAddonDeploy
```

Comment:
This will end up being the `rancher/hyperkube` image. Is this desirable or would a more streamlined image make sense?

Reply:
It will be the hyperkube image. The deploying by `rke-network-plugin-deploy-job` uses that image specifically calling `kubectl` in it which is exactly what it is being used for for Calico.
If there was a Kubectl image that was a better fit here we should use that.

An alternative to using Kubectl would be if there was a multiple stage deployment option that would allow 1st applying a CRD and then creation of an instance of the CR. Using Kubectl is only necessary to stage creation of the CR so it happens after the CRD is created.

Reply:
Is the goal to have `rke` itself create the CRD via the `rke-network-plugin-deploy-job` then down the line use the `kubectl` image to create the necessary CR's for the operator?

Reply:
The goal is to create a CRD and an instance of the CR.
With the CRD and the CR in the manifest, the `rke-network-plugin-deploy-job` will fail on the first attempt of the CR (because the CRD was not previously applied) and then pass on the 2nd attempt of the job. (So I guess that is another option.)
So I was trying to achieve applying the CRD and CR without any errors by having in the manifest the CRD and a kubectl pod that creates the CR.
If you want to see the manifest I've been testing check out https://github.com/rancher/kontainer-driver-metadata/pull/394